compiler/cgen.nim

```diff
@@ -1210,11 +1224,6 @@ proc newPreInitProc(m: BModule): BProc =
   # little hack so that unique temporaries are generated:
   result.labels = 100_000
 
-proc newPostInitProc(m: BModule): BProc =
-  result = newProc(nil, m)
-  # little hack so that unique temporaries are generated:
```

Comment:
Well I did write this 'little hack' because it's actually used code. I don't know why the tests don't cover it.

Reply:
I think that's just c&p from `newPreInitProc`, no code at all is emitted for this section.